src/Overloade2.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class Overloade2 {
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Введите целое число:");
+        int userInputInt = scanner.nextInt();
+        boolean isPositiveInt = isPositive(userInputInt);
+        System.out.println(isPositiveInt);
+
+        System.out.println("Введите число с плавающей точкой:");
+        double userInputDouble = scanner.nextDouble();
+        boolean isPositiveDouble = isPositive(userInputDouble);
+        System.out.println(isPositiveDouble);
+    }
+
+    public static boolean isPositive(int number) {
```

Comment:
Correct variant of the 3rd task implementation:

static boolean isPositive(int number) {
if (number > 0) {
System.out.println("Number is greater than zero");
return true;
} else {
System.out.println("Number is not less than zero");
return false;
}
}